src/PublicApiAnalyzers/UnitTests/AnnotatePublicApiAnalyzerTests.cs

```diff
@@ -246,6 +246,45 @@ public class C
             await VerifyCSharpAdditionalFileFixAsync(source, shippedText, unshippedText, newShippedApiText: shippedText, fixedUnshippedText);
         }
 
+        [Fact]
+        public async Task TestAddAndRemoveMembers_CSharp_Fix_WithAddedNullability_WithoutOblivious()
```

Comment:
💡 It would be good to have a test showing that the code fix annotates _shipped_ files in-place.

Reply:
Added